node/src/accountant/mod.rs

```diff
@@ -2723,6 +2735,40 @@ mod tests {
         );
     }
 
+    #[test]
+    #[should_panic(
+        expected = "internal error: entered unreachable code: Early stopped receivable scan \
+        was suggested to be followed up by the scan for PendingPayables, which is not supported though"
+    )]
+    fn start_scan_error_in_receivables_and_unexpected_instruction_from_early_stop_scan_rescheduling(
```

Comment:
I understand that you've written the code with utmost confidence that we'll never hit the `unreachable!()` statements and I respect that. At the same time, I feel concerned that the testing team/user might encounter this and will end up panicking their Node. To prevent such cases, I'd appreciate if we avoid such panicking statements in the future if possible. 🙏 

PS: This comments is not directly related to this case, but a generic statement.